File: EPRO.Library.v3.5/EPRO.Library.v3.5/Caching/SampleInitializedControl.cs

namespace ELibrary.Standard.Caching
{
    public class SampleInitializedControl : System.Windows.Forms.Control
    {

        /// <summary>
        /// Just a simple control that force the creation of handle under this calling thread
        /// </summary>
        /// <remarks></remarks>
        public SampleInitializedControl() : base()
        {
            CreateHandle();
        }
    }
}

File: EPRO.Library.v3.5/EPRO.Library.v3.5/Caching/SampleInitializedControl.vb

Namespace Caching
    Public Class SampleInitializedControl
        Inherits System.Windows.Forms.Control

        ''' <summary>
        ''' Just a simple control that force the creation of handle under this calling thread
        ''' </summary>
        ''' <remarks></remarks>
        Public Sub New()
            MyBase.New()
            Me.CreateHandle()

        End Sub

    End Class
End Namespace